packages/CollabCodeEdit-Tests.package/CCEPackageManagerTests.class/instance/setUp.st

```diff
@@ -1,5 +1,6 @@
 running
 setUp
-	packageManager := CCEPackageManager default.
-	testClass := Environment default classNamed: 'CCETestDummy'.
-	session := CCESessionMock new
\ No newline at end of file
+	super setUp.
+	session := CCESessionMock new.
+	session scope: self sessionScope.
+	packageManager listenForChangesFor: session
```

Comment:
I would suggest to not activate the Change listening until the ackCreateSessionMessage arrives